web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -324,119 +314,182 @@ Migrating your existing app to the new auth system is a two-step process:
 
 :::info Migrating a deployed app
 
-While going through these steps, we will focus first on doing the changes locally and your local development database.
+While going through these steps, we will focus first on doing the changes locally (including your local development database).
 
-Once we confirm everything works well locally, we will apply the same changes to the deployed app.
+Once we confirm everything works well locally, we will apply the same changes to the deployed app (including your production database).
 
 **We'll put extra info for migrating a deployed app in a box like this one.**
 :::
 
 #### 1. Migrate to the New Auth System
 
-You can follow these steps to migrate to the new auth system:
-
-1. Upgrade Wasp to the latest 0.12.X version (as described [above](https://discord.gg/rzdnErX)).
-
-
-1. Bump the version to `^0.12.0` in `main.wasp` (if you haven't already done this when [migrating the app to the new structure](#migrating-your-project-to-the-new-structure)).
-1. Create the new auth tables in the database by running:
+You can follow these steps to migrate to the new auth system (assuming you already migrated the project structure to 0.12, as described [above](#migrating-your-project-to-the-new-structure)):
 
+1. Ensure your **local development database is running**.
+1. **Do the schema migration** (create the new auth tables in the database) by running:
    ```bash
    wasp db migrate-dev
    ```
+You should see the new `Auth`, `AuthIdentity` and `Session` tables in your database. You can use the `wasp db studio` command to open the database in a GUI and verify the tables are there. At the moment, they will be empty (no data).
+1. **Do the data migration** (move existing users from the old auth system to the new one by filling the new auth tables in the database with their data):
 
-You should see the new `Auth`, `AuthIdentity` and `Session` tables in your database. You can use the `wasp db studio` command to open the database in a GUI and verify the tables are there.
+   1. **Implement your data migration function(s)** in e.g. `src/migrateToNewAuth.ts`.
```

Comment:
Main thing I did is indented data migration functions stuff under `Do the data migration`, before some of it was indented on one level, some was on the top level, this way it is clear when we start and when we are done with the data migration.